spec/models/serializers/work_queue/appeal_serializer_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+describe WorkQueue::AppealSerializer, :all_dbs do
+  context "when an appeal has a decision issue with a decision date in the future" do
+    let(:appeal) { create(:appeal, :decision_issue_with_future_date) }
+    subject { described_class.new(appeal, params: { user: user }) }
+
+    context "when a VSO user views an appeal" do
+      let(:user) { create(:user, :vso_role) }
+
+      context "when the restrict_poa_visibility feature toggle is on" do
+        before { FeatureToggle.enable!(:restrict_poa_visibility) }
+        after { FeatureToggle.enable!(:restrict_poa_visibility) }
```

Comment:
```suggestion
        after { FeatureToggle.disable!(:restrict_poa_visibility) }
```

(I think? Maybe we just don't need this?)

Reply:
Fixed, TY!